docs/wip/theme-shadowing.md

```diff
@@ -1,7 +1,82 @@
 
-## How to use theme shadowing on SFJ
-Explain how to overwrite components of themes
+>ℹ️ We strongly encourage you to check Gatsby's documentation and explore some [Shadowing](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/) examples.
+## How you'll use shadowing in the SFJ
+In the [Gatsby Conceptual Guides](https://www.gatsbyjs.com/docs/conceptual/plugins-themes-and-starters/), we have the following:
+
+*"Gatsby themes allow Gatsby site functionality to be packaged as a standalone product for others (and yourself!) to easily reuse."*
+
+In the SFJ context, this means that when using the [VTEX Store Components theme](https://github.com/vtex-sites/storecomponents.store), you will be able to use pre-configured functionalities, data sourcing, design, and other additional settings the VTEX theme includes.
+
+The VTEX Store Components theme packages already include a site structure, containing:
+
+- Home, Product, and Search pre-defined pages.
+- Pre-defined React components used as templates to build those pages.
+
+To customize your SFJ store and style it the way you want, we'll use Shadowing in the VTEX boilerplate theme. 
+
+>ℹ️ *When developing specific pages and components for your store, we suggest using the Store Components theme as a guide.*
+
+As stated in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/): 
+
+*"This feature allows users to replace a file in the `src` directory that is included in the webpack bundle with their own implementation."*
+
+That means that, with shadowing, you can adapt any file to suit your needs.
+
+For example, you can change styles to match your visual identity, or even a component functionality to meet your business rules.
+
+### What can I shadow?
+In the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/#any-source-file-is-shadowable) you'll learn that you can override (_shadow_) any source file, not just React components. In our example-store implementation, we shadowed this typescript [file](https://github.com/vtex/faststore/blob/master/packages/gatsby-plugin-i18n/src/i18n/en.ts) from `gatsby-plugin-i18n` to add our example-store expressions, transforming it into this [file](https://github.com/vtex-sites/storecomponents.store/blob/master/src/%40vtex/gatsby-plugin-i18n/i18n/en.ts). You can customize any file by creating a file in the same path as the theme you are using, example: `.../i18n/en.ts`, that way the Shadowing API will know that it should use your file, instead of the theme's file.
+
+:warning: Please note that since the Store Framework Jamstack is built using Gatsby, you are able to use any resource/concept/feature present in Gatsby. Therefore, all types of shadowing explained in the Gatsby documentation works perfectly when you are using the SFJ.
```

Comment:
```suggestion
Notice that you can customize any file by creating a corresponding one in the same path as the theme you are using (e.g., `.../i18n/en.ts`). This way, the Shadowing API will recognize which file it should use instead of the theme's default file.

>ℹ️ *Please note that since the SFJ is built using Gatsby, you can use any of its resources/concepts/features. Therefore, all shadowing techniques explained in the Gatsby documentation works perfectly when you are using the SFJ.*
```